### PR TITLE
[ADD] checkers: category-allowed

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**

### CODE REVIEW

**Comment by moylop260** [comment]
I like the new lint since the default values are empty so it could be enabled explicitly but disable implicitly 

So 👍 

@luisg123v could you review it, please?

File: src/pylint_odoo/checkers/odoo_addons.py [hunk 1]
```diff
@@ -512,6 +514,15 @@ class OdooAddons(OdooBaseChecker, BaseChecker):
                 "help": "List of valid odoo versions separated by a comma.",
             },
         ),
+        (
+            "category-allowed",
+            {
+                "type": "csv",
+                "metavar": "<comma separated values>",
+                "default": DFTL_CATEGORY_ALLOWED,
+                "help": "List of category allowed in manifest file, separated by a comma.",
```
Comment (luisg123v) [inline src/pylint_odoo/checkers/odoo_addons.py:523]:
```suggestion
                "help": "List of allowed categories in manifest file, separated by commas.",
```

File: src/pylint_odoo/checkers/odoo_addons.py [hunk 2]
```diff
@@ -1007,6 +1019,17 @@ def visit_dict(self, node):
         if license_str and license_str not in self.linter.config.license_allowed:
             self.add_message("license-allowed", node=manifest_keys_nodes.get("license") or node, args=(license_str,))
 
+        # Check category allowed
+        category_str = manifest_dict.get("category", None)
```
Comment (luisg123v) [inline src/pylint_odoo/checkers/odoo_addons.py:1023]: Superfluous default value for `get`

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by moylop260** [comment]
Also, could you add a unittest, please?

**Comment by luisg123v** [comment]
- Missing commit description.
- Where would the default allowed categories be set?

**Comment by moylop260** [comment]
@ThomasBinsfeld 

Will you able to re-take this work?

**Comment by sbidoul** [comment]
@ThomasBinsfeld gentle nudge

File: src/pylint_odoo/checkers/odoo_addons.py (hunk 1 above)
Comment (ThomasBinsfeld) [reply src/pylint_odoo/checkers/odoo_addons.py]: Fixed

File: src/pylint_odoo/checkers/odoo_addons.py (hunk 2 above)
Comment (ThomasBinsfeld) [reply src/pylint_odoo/checkers/odoo_addons.py]: Fixed

**Comment by ThomasBinsfeld** [comment]
Branch rebased and tests added :heavy_check_mark:

**Review by baimont — APPROVED** [review]

**Review by jdoutreloux — APPROVED** [review]

**Comment by ThomasBinsfeld** [comment]
Thanks!

**Comment by moylop260** [comment]
Thank you!

**Pull request merged**